libcnmc/cir_4_2015/F1.py

```diff
@@ -261,6 +261,9 @@ def consumer(self):
                     polissa = O.GiscedataPolissa.read(
                         polissa_id[0], fields_to_read, context_glob
                     )
+                    polissa_act = O.GiscedataPolissa.read(
+                        polissa_id[0], fields_to_read
+                    )
```

Comment:
Move this read bottom and only do it if is needed

Reply:
Done